array-manipulation-1/src/last.js

```diff
@@ -0,0 +1,9 @@
+/* exported last */
+// make storage for the last element of array
+// get last element of array and store it
+// give back location of storage for last element
+function last(array) {
+  var lastIndex = [];
```

Comment:
Making this variable with a default value of an empty array is unnecessary, you can just save the value you want in it when you declare the variable.